package.json

```diff
@@ -84,7 +84,7 @@
     "eslint-plugin-react": "^7.11.1",
     "go-ipfs-dep": "~0.4.18",
     "interface-ipfs-core": "~0.96.0",
-    "ipfsd-ctl": "~0.40.0",
+    "ipfsd-ctl": "github:ipfs/js-ipfsd-ctl#feat/support-profile",
```

Comment:
what's missing to have this shipped? :D

Reply:
Nothing, I just didn't PR yet!

Reply:
I actually missed what this “test profile” was in the beginning. Super rad to see this go-IPFS feature.

Would love to see this on interop tests (and see if it doesn’t break my router anymore).

Reply:
I should have explained better! For everyone else, you can pass a "profile" option to `ipfs init`:

> 'test' - Reduces external interference of IPFS daemon, this is useful when using the daemon in test environments.
> https://docs.ipfs.io/reference/api/cli/#ipfs-init